napari/settings/_tests/test_settings.py

```diff
@@ -300,6 +301,41 @@ def test_get_settings(tmp_path):
     assert str(s.config_path) == str(p)
 
 
+def test_get_prev_ver_settings(monkeypatch, tmp_path):
+    """This test tests fallback to previous version settings"""
```

Comment:
Nit: feel free to ignore, but consider changing the name of the test to `test_fallback_to_previous_version_settings`. That way you save a line, save some repetition, and also avoid some abbreviation in the function name.